panther

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env php
+<?php
+
+/*
+ * This file is part of the Panther project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use Psy\Shell;
+use Symfony\Component\Panther\Client;
+use Symfony\Component\Panther\PantherTestCaseTrait;
+
+require file_exists($autoload = __DIR__.'/vendor/autoload.php') ? $autoload : __DIR__.'/../../autoload.php';
+
+if (file_exists($psyshFile = '/usr/local/bin/psysh')) {
```

Comment:
Should be configurable isn’t it?

Reply:
It's the default path [suggested by PsySH](https://github.com/bobthecow/psysh/wiki/Installation#-installation).
If you really want a configuration for it, how does it look like ? An env var like `PANTHER_PSYSH_BIN` ?

Reply:
Yes and env var would be nice.

Reply:
Done.